### PR TITLE
Improve UI for finished duels 

### PR DESCRIPTION
**Pull request by valentinperignon**
## Changes

- [x] Home page

Light             |  Dark
:-------------------------:|:-------------------------:
Screen Shot 2021-03-17 at 09 01 14 | Screen Shot 2021-03-17 at 09 01 34

- [x] Duel overview

Light             |  Dark
:-------------------------:|:-------------------------:
Screen Shot 2021-03-17 at 09 01 56 |  Screen Shot 2021-03-17 at 09 01 48

### CODE REVIEW

File: client/src/pages/DuelOverview.js [hunk 1]
```diff
@@ -88,7 +89,13 @@ const DuelOverview = ({
     return <PageError message="Échec du chargement du duel" />;
   }
 
-  const { currentUserScore, opponentScore, opponent, inProgress } = data;
+  const { currentUserScore, opponentScore, opponent, inProgress, finishedDate, TTL } = data;
+
+  const remainingTimeBefeoreRemove = (function () {
```
Comment (nathanaelhoun) [inline client/src/pages/DuelOverview.js:94]:
```suggestion
  const remainingTimeBeforeRemove = (function () {
```

File: client/src/pages/DuelOverview.js [hunk 2]
```diff
@@ -109,12 +116,18 @@ const DuelOverview = ({
...
+        <>
+          <DuelResults
+            user={currentUser}
+            opponent={opponent}
+            score={currentUserScore}
+            opponentScore={opponentScore}
+          />
+          <p id="TTL">
+            Le duel sera supprimé dans {remainingTimeBefeoreRemove}{" "}
+            <Plural word="jour" count={remainingTimeBefeoreRemove} />
```
Comment (nathanaelhoun) [inline client/src/pages/DuelOverview.js:128]:
```suggestion
            Le duel sera supprimé dans {remainingTimeBeforeRemove}{" "}
            <Plural word="jour" count={remainingTimeBeforeRemove} />
```

File: client/src/utils/queryDuels.js [hunk 3]
```diff
@@ -57,9 +57,11 @@ export function makeGetDuelDetails(duelId) {
       opponent: users[opponent],
       currentUserScore: duel.userScore,
       opponentScore: duel.opponentScore,
-      // eslint-disable-next-line eqeqeq
+      // eslint-disable-next-line
```
Comment (nathanaelhoun) [inline client/src/utils/queryDuels.js:60]: Why this change?

File: server/controllers/duels.js [hunk 4]
```diff
@@ -391,11 +391,20 @@ function createShuffledQuestionTypesArray() {
...
     return null;
   }
-  return formatDuel(res[0], username);
+
+  const TTL = await (async function () {
+    const res = await queryPromise(
+      "SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';"
+    );
+    return res[0].value;
+  })();
```
Comment (FrancoisPog) [inline server/controllers/duels.js:403]: You could use something like this
```suggestion
  const TTL = (await queryPromise( "SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';"
    ))[0].value;


```

Comment (nathanaelhoun) [inline server/controllers/duels.js:403]: It it possible to have the two calls running in parallel with an `Promise.all()` to avoid increasing the time to request a duel?

**Review by FrancoisPog — APPROVED** [review]
LGTM !

**Review by nathanaelhoun — APPROVED** [review]
Looking really good!

Nitpick: Can we have different icons for the "only x hours remains to complete the duel" and "the duel was finished on xxx" in HomePage? For the latter we can use the `Counter-clockwise clock` for example.

File: server/controllers/duels.js (hunk 4 above)
Comment (valentinperignon) [reply server/controllers/duels.js:403]: Trop smart as we say in France

File: client/src/utils/queryDuels.js (hunk 3 above)
Comment (valentinperignon) [reply client/src/utils/queryDuels.js:60]: Oops

File: server/controllers/duels.js (hunk 4 above)
Comment (valentinperignon) [reply server/controllers/duels.js]: Let's do this

Comment (valentinperignon) [reply server/controllers/duels.js]: But after the réunion

File: client/src/utils/queryDuels.js (hunk 3 above)
Comment (nathanaelhoun) [reply client/src/utils/queryDuels.js:60]: eqeqeq

File: server/controllers/duels.js (hunk 4 above)
Comment (FrancoisPog) [reply server/controllers/duels.js]: You can also do it in just one query!

Comment (valentinperignon) [reply server/controllers/duels.js]: That's the query you already commented @FrancoisPog :)

Comment (FrancoisPog) [reply server/controllers/duels.js]: Yep but maybe this is even better 
```js
const res = await queryPromise("CALL getDuel(?,?); SELECT `value` FROM `server_informations` WHERE `key` = 'config_duel_rounds_per_duel';");
```

Comment (valentinperignon) [reply server/controllers/duels.js]: Oh okay! Sure
